functions/deploy/main.py

```diff
@@ -52,6 +58,11 @@ def deploy_commit_to_door43(self, build_log_key):
         output_dir = tempfile.mkdtemp(prefix='output_')
         template_dir = tempfile.mkdtemp(prefix='template_')
 
+        # keep track of temp folders for easier cleanup
+        self.source_dir = source_dir
+        self.output_dir = output_dir
+        self.template_dir = template_dir
```

Comment:
@phillip-hopper - These changes were for unit testing cleanup.  I probably should just add cleanup of temp folders to deploy method.